src/components/inputs/Selector.tsx

```diff
@@ -169,6 +172,7 @@ export default function Selector({
           }}
           openOnFocus
           onKeyDown={handleKeyDown}
+          ListboxProps={{ sx: { maxHeight: { xs: '175px', sm: '299px' } } }}
```

Comment:
On the design it seems to be 291px... can you please confirm how this value was reached

If the figma value causes any visual inconsistencies for our use case we should use this value, else we can be on the safe side and decrease it by a few pixels :stuck_out_tongue: 

Ref: https://www.figma.com/design/Z9ONck5HGIqNDszxj5YM0G/Custom-%26-Marketplace-Apps?node-id=1554-11774&m=dev